src/index.html

```diff
@@ -1,27 +1,57 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>title</title>
-  <link rel="stylesheet" href="css/style.ss" type="text/css">
-  </head>
-  <body>
-      <div class="error" style="color:pink">
-          Looks like the styleguide is not being implemented correctly..
-      </div>
-      <div class="content">
-          <h1 class="title">
-              Hmm...
-          </h1>
-          <p class="text">
-              Something is wrong... <br> 
-              Looks like the Javascript code is not being executed. <br>
-              Fix me please.
-          </p>
-      </div>
-      <div class="success" style="color:white; position:absolute; top:580px">
-            Woohoo, CSS is working!
+    <title>VI Onboarding Fundamentals II</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="card">
+        <div class="card__content">
+            <h1 class="card__title">
+                A wonderful serenity has taken possesion
+            </h1>
+            <p class="card__text">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            </p>
+            <a href="javascript:" class="card__button">Link button</a>
         </div>
-      <script type="module" src="js/mayn.js"></script>
-  </body>
-</html> 
\ No newline at end of file
+        <div class="card__imgBox">
+            <img src="https://dummyimage.com/800x600/c7c7c7/fff" alt="Image" class="card__img">
+        </div>
+    </div>
+
+    <div class="card card--coloured">
```

Comment:
It would be worth thinking about extracting the '--coloured' modifier to the headline and the button instead of the whole component, as it only affects them. That's a coding choice though, both ways are valid.